config/galaxy.yml.sample

```diff
@@ -812,6 +812,11 @@ galaxy:
   # documentation for the corresponding nginx configuration.
   #nginx_upload_job_files_path: false
 
+  # Galaxy can upload user files in chunks without using nginx. Enable
```

Comment:
This documentation should first go in lib/galaxy/webapps/galaxy/config_schema.yml , while the changes here are generated with `make config-rebuild`